src/Sharpliner/AzureDevOps/TemplateDefinition.cs

```diff
@@ -220,7 +220,7 @@ public abstract class TemplateDefinition<T> : TemplateDefinition, ISharplinerDef
     /// Returns the definition of the template.
     /// </summary>
     [DisallowNull]
-    public abstract ConditionedList<T> Definition { get; }
+    public abstract T Definition { get; }
```

Comment:
Why did this change?

Reply:
I added `TemplateDefinitionBase<T>` which is the base class for `TemplateDefinition<ConditionedList<T>>`

this is in order to support templates of `extends` which contain two properties, `parameters` & `extends` but the `extends` is not a list, just a single object

Reply:
I will need to have a closer look to understand if it should be `Conditioned<T>`